### ISSUE TITLE
control empty role of boundaries

### ISSUE THREAD

**Issue by Mahabarata** [openstreetmap/iD#9127]
### Description

Hi,

currently, when a user creates a multipolygon and forgets to add a role to one or more members, iD shows a warning. This is a good thing if we read the wiki : https://wiki.openstreetmap.org/wiki/Relation:multipolygon = a member with no role is indicated as deprecated.

In the wiki, we find the same rule for boundaries : https://wiki.openstreetmap.org/wiki/Relation:boundary = a member with no role is also indicated as deprecated.

But ID shows no warning when a mapper forgets a role and sends to osm database a boundary relation with one or more members with an empty role.

Could you add a control on empty roles of boundary relations like the one for multipolygon relations ?

For your informations, there were more than 17,000 boundary relations with empty roles a few month ago. Yesterday, zero on a way. Today 5 relations with empty roles on a way = all have been created with ID.

Best regards

**Comment by Mahabarata** [comment]
Some stats :
* between the 30 May and the 7 July, 124 boundary relations with at least one member with an empty role have been created : 109 with iD, 11 with JOSM, 3 with RapiD and 1 with Vespucci
* for these 124 boundaries, there were 293 members without role

List of these 124 relations, if needed :
[boundary empty2_res.csv](https://github.com/openstreetmap/iD/files/9114607/boundary.empty2_res.csv)

**Comment by Mahabarata** [comment]
Some stats :

* between the 8 July (2022) and the 14 March (2023), 6678 nodes/ways/relations are in a boundary without a role (according to taginfo) : some of these elements can be counted several times if in seveval boundary relations
* 6107 unique elements are at least in a boundary with no role : 3397 were created by JOSM, 2257 by iD

**Comment by tyrasd** [comment]
Good idea. The boundary relations do have a small difference compared to multipolygon relations: in addition to outer/inner, there are also the roles `admin_centre` and `label` (for nodes) and `subarea` (for relation members) which are allowed.

**Comment by matkoniecz** [comment]
@tyrasd maybe better to finish https://github.com/openstreetmap/id-tagging-schema/pull/1538 and solve it in general?
